working-groups/learning/minutes/agenda.md

```diff
@@ -0,0 +1,5 @@
+# WG-Learning future meeting agenda
```

Comment:
I'd use a different name for this file that explains better what the title says. Nothing comes to my mind to be honest, maybe a native speaker can help here :).

Reply:
"Agenda" is "a list of items to be discussed at a formal meeting". Perhaps `future_discussion_topics.md` ?